srcpkgs/grex/patches/force-std-indexmap.patch

```diff
@@ -0,0 +1,13 @@
+# Drop when using petgraph version greater than 0.6.2
+diff --git a/Cargo.toml b/Cargo.toml
+index 3aa40a65cb..f5b93e8978 100644
+--- a/Cargo.toml
++++ b/Cargo.toml
+@@ -38,6 +38,7 @@ itertools = "0.10.5"
+ lazy_static = "1.4.0"
+ ndarray = "0.15.6"
+ petgraph = {version = "0.6.2", default-features = false, features = ["stable_graph"]}
++indexmap = {version = "1.7", features = ["std"] }
```

Comment:
I assume this is the autocfg issue? could this be done with `cargo update --package autocfg:X.Y.Z --precise A.B.C` like most of the other packages with indexmap/autocfg issues?

Reply:
Thanks, that fixes it without having to add a patch.